model/ftn/w3agcmmd.ftn

```diff
@@ -132,7 +132,7 @@
          ! Ocean sea surface current (m.s-1) (v-component)
          ! ---------------------------------------------------------------------
          IF (SND_FLD(IB_DO)%CL_FIELD_NAME == 'WW3_WSSV') THEN
-            TMP(1:NSEALM) = 0.0
+            TMP(1:NSEAL) = 0.0
             DO JSEA=1, NSEAL
```

Comment:
not even sure this section works well since TMP array is filled after the test on UNDEF
Maybe you should also rewrite it as : 
WHERE(CY(1:NSEAL) /= UNDEF) TMP(1:NSEAL)=CY(1:NSEAL)